pkg/settings/settings.go

```diff
@@ -70,14 +72,20 @@ func (r *ControllerSettings) Load() error {
 // Get positive integer limit from the environment
 // using the specified variable name and default.
 func getEnvLimit(name string, def int) (int, error) {
+	return getEnvLimitMinimal(name, def, 1)
+}
+
+// Get an integer limit from the environment
+// using the specified variable name and default.
+func getEnvLimitMinimal(name string, def, minimum int) (int, error) {
```

Comment:
`getEnvLimitMinimal` doesn't indicate what the 'Minimal' part stands for clearly
how about renaming `getEnvLimit` to `getPositiveEnvLimit` and introduce another `getNonNegativeEnvLimit`?